jmespath-core/src/main/java/io/burt/jmespath/node/JsonLiteralNode.java

```diff
@@ -3,35 +3,33 @@
 import io.burt.jmespath.Adapter;
 
 public class JsonLiteralNode<T> extends Node<T> {
-  private final String raw;
-  private final T tree;
+  private final String rawJson;
+  private final T json;
```

Comment:
I think the name `tree` was better than `json`. JSON is the serialized format, and the parsed tree is no longer JSON. Maybe `value`?